src/Components/SigTable/SigTable.js

```diff
@@ -162,9 +162,9 @@ const SigTable = () => {
                 rowId: key,
                 isOpen: false,
                 cells: [
-                    { title: <Link to={`/signatures/${sig.name}`}>{sig.name}</Link> },
+                    { title: <Link to={`${BASE_PATH}/signatures/${sig.name}`}>{sig.name}</Link> },
```

Comment:
Using the `InsightsLink`[0] component here would allow avoiding the path change here. 

[0] https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/doc/insights_link.md

Reply:
Done!